package.json

```diff
@@ -169,7 +169,7 @@
   "scripts": {
     "start": "vite",
     "codecheck": "tsc --noEmit",
-    "test": "jest --noStackTrace",
+    "test": "jest",
```

Comment:
Last time I checked, the stack trace was quite inaccurate. Is this no longer the case now?